mooncake-integration/transfer_engine/transfer_engine_py.cpp

```diff
@@ -688,4 +742,9 @@ PYBIND11_MODULE(engine, m) {
                  &TransferEnginePy::getFirstBufferAddress);
 
     adaptor_cls.attr("TransferOpcode") = transfer_opcode;
+
+    // Add coro_rpc_interface as a submodule
+    auto coro_rpc_submodule = m.def_submodule(
+        "coro_rpc_interface", "CoroRPC interface for communication");
+    bind_coro_rpc_interface(coro_rpc_submodule);
```

Comment:
`coro_rpc_interface` should not be a submodule of `transfer_engine`. It is misplaced. Use `two_sided` or a similar alternative instead of `coro_rpc`.